gnovm/stdlibs/os/proc.go

```diff
@@ -0,0 +1,10 @@
+package os
+
+// Args hold the command-line arguments, starting with the program name.
+var Args []string
+
+func init() {
+	Args = runtime_args()
+}
+
+func runtime_args() []string // in package runtime
```

Comment:
Second option.

I suggest: VMKeeper to be the preferred "frontend" for configuring the VM for advanced use cases like full context, multi-transaction features etc, while the raw gno.Machine remains suitable for single-test focused tasks.

Related with #1016.
After #1016 will be finished, I plan to migrate most of the `gno` CLI to it instead of `gno.Machine`, especially the REPL.

Reply:
Sounds good.